t/parse_assignments.pl

```diff
@@ -0,0 +1,55 @@
+use Test::More;
+use Data::Dumper;
```

Comment:
Probably all distributions ship this core module but we might wanna add it anyway to the test requirements just to make sure none of these drop them in the future.

Reply:
Done.